1. values added: 20
2. final result: 20
3. values added: 20
4. The code returns an error because result is not defined outside of the function sumValues due to let provides the variable with block scope
5. 